### PR TITLE
Remove useless config elements

### PR DESCRIPTION
**Pull request by benzekrimaha**
Issue : BB-520

### CODE REVIEW

File: lib/Config.js
Comment (ghost) [inline lib/Config.js:1]: Just saw that, but do we still use the 8.5 branch? I see 8.6 in Zenko 2.6

Comment (ghost) [reply lib/Config.js:1]: (affects the other pr, although it's harmless, maybe now is the right time to archive the 8.5 branch, if not used anymore)

File: lib/Config.js [hunk 1]
```diff
@@ -106,14 +106,6 @@ class Config extends EventEmitter {
                 { host: '127.0.0.1', port: 6379 });
         }
 
-        if (parsedConfig.localCache) {
```
Comment (ghost) [inline lib/Config.js:116]: Should be removed from the json file as well, maybe also from the operator (https://github.com/scality/zenko-operator/blob/development/1.6/pkg/controller/zenko/reconcile_backbeat_config.go#L982)

File: lib/Config.js
Comment (benzekrimaha) [reply lib/Config.js:1]: yes , going to target the 8.7 instead in all PRs

File: lib/Config.js (hunk 1 above)
Comment (benzekrimaha) [reply lib/Config.js:116]: done here : https://github.com/scality/backbeat/pull/2534/commits/1ad8a8219e5107e9c1340432457538387a5833ec#diff-10001999fea4d2498276120abf7902b81510153910d8314c7493ddafa6e19e09L324-L331, for the operator i'm going to check then create a ticket if necessary

Comment (benzekrimaha) [reply lib/Config.js:116]: ticket created here : https://scality.atlassian.net/browse/ZKOP-408

**Review by ghost — APPROVED** [review]

**Review by francoisferrand — APPROVED** [review]

**Pull request merged**